### PR TITLE
(201) Use 'midday' instead of '12pm'

### PR DESCRIPTION
**Pull request by dgmstuart**
As per the gov.uk style guide:
https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#times

Also use 'to' instead of dashes in time ranges

### CODE REVIEW

**Comment by dgmstuart** [comment]
rebased

**Comment by leeky** [comment]
@dgmstuart Should we do the same when listing available appointments? Given the ongoing fixes to the API endpoints this feels like a separate card perhaps..?

File: app/presenters/appointment_presenter.rb [hunk 1]
```diff
@@ -59,6 +59,7 @@ def end_time
   end
 
   def friendly_time(time)
+    return 'midday' if time == time.midday
```
Comment (leeky) [inline app/presenters/appointment_presenter.rb:62]: Rails has *almost* too many convenience methods :+1:

**Review by leeky — APPROVED** [review]
LGTM!

My only comment was whether we should apply these changes in appointment booking and static pages here or create new cards..?

**Comment by dgmstuart** [comment]
@leeky Can this be merged? Were there some places I missed?

**Comment by leeky** [comment]
@dgmstuart There are a few places static pages which have time ranges separated by `&ndash;` instead of ` to `. I'll add these, rebase and merge this in :+1:

**Pull request merged**